engine/runtime/src/boxed.rs

```diff
@@ -145,10 +146,40 @@ impl<T: Bytes> Boxed<T> {
         if unsafe { sodium_init() == -1 } {
             panic!("Failed to initialize libsodium")
         }
+        let mut rng = random::thread_rng();
+
+        // TESTING
```

Comment:
Are the comments part of the "virtual" allocation of more fragments, to indicate a block with some "extra work" eg. allocate some data to create space between the shards?

Reply:
I didn't notice that this code was added. I guess this is used to create space between the shards. But as it is, this is used even for normal memory allocation,  I will check it it is necessary